opal/resource.go

```diff
@@ -180,6 +187,103 @@ func resourceResource() *schema.Resource {
 				Type:        schema.TypeBool,
 				Optional:    true,
 				Default:     true,
+				Deprecated:  "Use request_configurations_list instead.",
```

Comment:
in the deprecated comments, it refers to `request_configurations_list`, but I can't find that - do you mean `request_configuration`?